.github/workflows/checkout-cd-inte.yml

```diff
@@ -0,0 +1,86 @@
+name: Module checkout CD
+
+on:
+  push:
+    branches:
+      - dev
+
+env:
+  MODULE_NAME: ps_checkout
+  GCLOUD_TOKEN_PATH: ./token.json
+
+jobs:
+  checkout_cd:
+    name: Module Checkout continuous deployment
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    if: contains(github.event.pull_request.labels.*.name, 'quality insurance needed')
```

Comment:
Is this check going to work? If you push to `dev`, there shouldn't be any more `pull_request` array to rely on?

Reply:
Thx for the review ! I will change the trigger to be pull_request.